cookies/views.py

```diff
@@ -1049,3 +1049,37 @@ def resource_merge(request):
     })
     template = loader.get_template('resource_merge.html')
     return HttpResponse(template.render(context))
+
+@login_required
+def create_collection(request):
+    """
+    Curator can add collection
+    """
+    context = RequestContext(request, {})
+
+    template = loader.get_template('create_collection.html')
+
+    if request.method == 'GET':
+        form = UserAddCollectionForm()
+    if request.method == 'POST':
+        form = UserAddCollectionForm(request.POST)
+        if form.is_valid():
+            collection = Collection.objects.create(**{
```

Comment:
See my comments above about using a ``ModelForm`` -- the pattern in this view is so ubiquitous (creating a model instance from form data) that Django will do it all for you. But you have to use a ``ModelForm``.